gnomad_qc/v3/utils.py

```diff
@@ -0,0 +1,25 @@
+import hail as hl
+
+
+def hom_alt_depletion_fix(
```

Comment:
Sorry, approval assumes this thread will be resolved at a later date: https://atgu.slack.com/archives/CNNTF8Z46/p1620401614102100 maybe add a TODO?